frontend/src/components/faq/faqs.css

```diff
@@ -0,0 +1,78 @@
+.faq-image{
+    width:100%;
+    height:90vh;
+    background: rgba(0, 0, 0, 0.3);
+}
+
+
+.faqs-top-section{
+    position:relative;
+}
+.faqs-top-section h1{
+    position: absolute;
+    top:20px;
+    right:200px;
+    color:black;
+    font-weight: 900;
+    font-family: 'Futura LT Book';
+    font-size:50px;
+    text-shadow: 2px 2px black;
+}
+
+.faqs-section{
+    height: 90vh;
+    background-color: white;
+}
+
+.faqs-title-text{
+    text-align: center;
+    font-size: 1.5rem;
+    color: #016795;
+}
+
+.faqs-title-subtext{
+    font-weight: bold;
+    text-align: center;
+    color: #1B2431;
+}
+
+.faq-block,#panel1bh-header,#panel2bh-header,#panel3bh-header,#panel4bh-header{
+
+    background-color:  #d9ecf2;
+}
+
+#panel1bh-content,#panel2bh-content, #panel3bh-content, #panel4bh-content{
+    background-color: #f1f9f9;
+}  
+
+.faq-block{
+   
+    box-shadow: 0 4px 2px 0 rgba(0, 0, 0, 0.19);
+}
+
+
+.faq-question{
+    font-weight: 600; 
+    font-size: 1rem; 
+}
+.MuiPaper-root.Mui-expanded, .MuiPaper-root::after{
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 4px 5px 0 rgba(0, 0, 0, 0.19);
+}
+
+#panel1bh-header.Mui-expanded, #panel2bh-header.Mui-expanded, #panel3bh-header.Mui-expanded, #panel4bh-header.Mui-expanded{
+    background-color: #eff8ff   ;
+   
+}
+.container{
```

Comment:
This class is conflicting somehow. As it's changing the styles on `about-us` and `motive` section of the home page. Please fix it.

![Screenshot from 2020-12-30 13-09-39](https://user-images.githubusercontent.com/44888949/103337191-84502600-4aa0-11eb-942f-49cf429900c8.png)